services/app-api/utils/formTemplates/formTemplates.ts

```diff
@@ -244,3 +270,43 @@ export function getValidationFromFormTemplate(reportJson: ReportJson) {
 export function getPossibleFieldsFromFormTemplate(reportJson: ReportJson) {
   return Object.keys(getValidationFromFormTemplate(reportJson));
 }
+
+const routesToIncludeInPCCM = {
+  "A: Program Information": [
+    "Point of Contact",
+    "Reporting Period",
+    "Add Plans",
+  ],
+  "B: State-Level Indicators": ["I: Program Characteristics"],
+  "C: Program-Level Indicators": ["I: Program Characteristics"],
+  "D: Plan-Level Indicators": ["I: Program Characteristics", "VIII: Sanctions"],
+  "Review & Submit": [],
+} as { [key: string]: string[] };
+
+const entitiesToIncludeInPCCM = ["plans", "sanctions"];
+
+export const generatePCCMTemplate = (originalReportTemplate: any) => {
+  const reportTemplate = structuredClone(originalReportTemplate);
```

Comment:
this structuredClone let's us copy the mcpar template without modifying it in memory, so that we can create MCPARs and PCCMs and they will always be based on the full MCPAR